cmd/bootnode/main.go

```diff
@@ -11,13 +11,31 @@ import (
 	"github.com/ethereum/go-ethereum/p2p/discv5"
 )
 
+type nursery []*discv5.Node
+
+func (f *nursery) String() string {
+	return "discv5 nursery nodes"
+}
+
+// Set parses single topic.
+func (f *nursery) Set(value string) error {
+	n, err := discv5.ParseNode(value)
+	if err != nil {
+		return err
+	}
+	*f = append(*f, n)
+	return nil
+}
+
 func main() {
 	var (
 		listenAddr  = flag.String("addr", ":30301", "listen address")
 		nodeKeyFile = flag.String("nodekey", "", "private key filename")
 		verbosity   = flag.Int("verbosity", int(log.LvlInfo), "log verbosity (0-9)")
 		vmodule     = flag.String("vmodule", "", "log verbosity pattern")
+		bootnodes   = nursery{}
 	)
+	flag.Var(&bootnodes, "n", "nursery nodes for this bootnode")
```

Comment:
it would be nice to have some context on the flag info, like:

`nursery nodes for this bootnode. These nodes are used to connect to the network if the table is empty and there are no known nodes in the database.`

[reference](https://github.com/ethereum/go-ethereum/blob/master/p2p/discv5/net.go#L196)

Reply:
added